### DISCUSSION TITLE
Wave Styling

### DISCUSSION THREAD

**Discussion by shrix1** [katspaugh/wavesurfer.js#3236] [Ideas]
Is there any Way to style a Nice Looking wave like youtube .. If Possible can someone help me on that

**Comment by katspaugh** [comment]
What's a "Nice Looking wave like youtube"? And what have you tried so far?

**Comment by shrix1** [comment]
<img width="996" alt="Screenshot 2023-10-06 at 5 17 45 PM" src="https://github.com/katspaugh/wavesurfer.js/assets/92677078/169ff319-2e46-4f75-9569-c1f1188e9a8b">
Like this mate

**Reply by katspaugh** [reply]
You can start with `barAlign: 'bottom'`.
https://wavesurfer.xyz/examples/?all-options.js

**Reply by shrix1** [reply]
Could you please provide the renderFunction to render waveform like the image I sent?. I'm struggling since last 2 weeks. 
Thanks for the quick response.

**Comment by shrix1** [comment]
Thanks
